src/partials/head-last.hbs

```diff
@@ -1,7 +1,7 @@
 <link rel="stylesheet" href="{{{uiRootPath}}}/css/site.css">
 <script src="{{{uiRootPath}}}/js/vendor/jquery.js"></script>
 {{#with env.OPTANON_SCRIPT_URL}}
-<script src="{{this}}"></script>
+<script src="{{this}}" data-domain-script="{{env.OPTANON_SCRIPT_DATA_DOMAIN_SCRIPT}}"></script>
```

Comment:
Since there are now two environment variables for optanon, I recommend rewriting this as follows:

```html
{{#if env.OPTANON_SCRIPT_URL}}
<script src="{{env.OPTANON_SCRIPT_URL}}" data-domain-script="{{env.OPTANON_SCRIPT_DATA_DOMAIN_SCRIPT}}"></script>
{{/if}}
```

The `with` requires a different syntax when accessing two different properties.